lib/gates_cirq.h

```diff
@@ -1073,6 +1160,9 @@ struct FSimGate {
 
 // Gates from cirq/ops/matrix_gates.py:
 
+/**
+ * A unitary one-qubit gate defined entirely by its matrix.
```

Comment:
Strictly  speaking, MatrixGate1 and MatrixGate2 gates do not have to be unitary in general.

Reply:
Done.